build/cqs.tsv

```diff
@@ -173,6 +173,7 @@ com.datastax.cassandra:cassandra-driver-core	3.0.0	provided
 com.datastax.cassandra:cassandra-driver-mapping	3.0.0	provided
 javax.media:jai_codec	1.1.3	provided
 javax.media:jai_core	1.1.3	provided
+jline:jline	1.0	provided
```

Comment:
I don't think we can allow jline here - it's been rejected already. Can you figure out what dependency it's coming from and exclude it in the pom?